processor/transformprocessor/internal/metrics/func_convert_summary_count_val_to_sum_test.go

```diff
@@ -17,101 +17,19 @@ package metrics
 import (
 	"testing"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/contexts/tqlmetrics"
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/pmetric"
-
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/contexts/tqlmetrics"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql/tqltest"
 )
 
-func getTestSummaryMetric() pmetric.Metric {
-	metricInput := pmetric.NewMetric()
-	metricInput.SetDataType(pmetric.MetricDataTypeSummary)
-	metricInput.SetName("summary_metric")
-	input := metricInput.Summary().DataPoints().AppendEmpty()
-	input.SetCount(100)
-	input.SetSum(12.34)
-
-	qVal1 := input.QuantileValues().AppendEmpty()
-	qVal1.SetValue(1)
-	qVal1.SetQuantile(.99)
-
-	qVal2 := input.QuantileValues().AppendEmpty()
-	qVal2.SetValue(2)
-	qVal2.SetQuantile(.95)
-
-	qVal3 := input.QuantileValues().AppendEmpty()
-	qVal3.SetValue(3)
-	qVal3.SetQuantile(.50)
-
-	attrs := getTestAttributes()
-	attrs.CopyTo(input.Attributes())
-	return metricInput
-}
-
-func getTestGaugeMetric() pmetric.Metric {
-	metricInput := pmetric.NewMetric()
-	metricInput.SetDataType(pmetric.MetricDataTypeGauge)
-	metricInput.SetName("gauge_metric")
-	input := metricInput.Gauge().DataPoints().AppendEmpty()
-	input.SetIntVal(12)
-
-	attrs := getTestAttributes()
-	attrs.CopyTo(input.Attributes())
-	return metricInput
-}
-
-func getTestAttributes() pcommon.Map {
-	attrs := pcommon.NewMap()
-	attrs.InsertString("test", "hello world")
-	attrs.InsertInt("test2", 3)
-	attrs.InsertBool("test3", true)
-	return attrs
-}
-
-func summaryTest(tests []summaryTestCase, t *testing.T) {
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			actualMetrics := pmetric.NewMetricSlice()
-			tt.input.CopyTo(actualMetrics.AppendEmpty())
-
-			evaluate, err := tql.NewFunctionCall(tt.inv, Functions(), tqlmetrics.ParsePath, tqlmetrics.ParseEnum)
-			assert.NoError(t, err)
-
-			evaluate(tqlmetrics.NewTransformContext(pmetric.NewNumberDataPoint(), tt.input, actualMetrics, pcommon.NewInstrumentationScope(), pcommon.NewResource()))
-
-			expected := pmetric.NewMetricSlice()
-			tt.want(expected)
-			assert.Equal(t, expected, actualMetrics)
-		})
-	}
-}
-
-type summaryTestCase struct {
-	name  string
-	input pmetric.Metric
-	inv   tql.Invocation
-	want  func(pmetric.MetricSlice)
-}
-
-func Test_ConvertSummarySumValToSum(t *testing.T) {
```

Comment:
These 2 test files shouldn't been using the function directly but were still calling NewFunctionCall.  They are now in line with how we test other functions.